cli/lib/cube.ts

```diff
@@ -33,11 +35,25 @@ export function expirePreviousVersions(this: Pick<Context, 'variables' | 'log'>)
     password: this.variables.get('publish-graph-store-password'),
   })
 
+  const job = this.variables.get('publish-job')
+  let onlyDraftCubes = sparql``
+  if (Draft.equals(job?.status)) {
+    onlyDraftCubes = sparql`
+      ?cube ${schema.creativeWorkStatus} ?status .
+
+      filter (
+        ?status ${IN(Draft, $rdf.namedNode('https://ld.admin.ch/definedTerm/CreativeWorkStatus/Draft'))}
+      )
+    `
+  }
+
   return CONSTRUCT`
     ?cube ${schema.expires} ${timestamp} .
   `.WHERE`
     ${baseCube} ${schema.hasPart} ?cube .
 
+    ${onlyDraftCubes}
+
```

Comment:
We are now also using this function to unlist cubes and I suspect this change would break that.
Should we create a separate function to unlist cubes?

Reply:
No need, it does not break. The filter is only added for publish job with status `Draft`

Reply:
Ok, I got confused by `const job = this.variables.get('publish-job')`. Maybe worth adding a comment.